src/autocomplete_abstract.coffee

```diff
@@ -326,7 +330,12 @@ class AutocompleteAbstract
         @list
 
     handleEmptyList: ->
-        @close()
+        if @options.showNoResultsNotice
+            noResultNotice = $ '<div class=empty-list>No Result Found…</div>'
+            noResultNotice.on 'click', (e) => @close()
```

Comment:
There were some undesired behaviour. For instance, if you click outside/blur from the input, the dropdown disappeared, but not the notice. It would be fine, but in our example, the input shrinks and gets another visual design. Perhaps it would be more suitable to handle this outside of reform…